engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js

```diff
@@ -25,7 +25,12 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsController
             loading: true
         };
 
-        $scope.host = Host.get({id: $scope.$stateParams.hostId}, function () {
+        $scope.host = Host.get({id: $scope.$stateParams.hostId}, function (host) {
+            if (host.hasSubscription()) {
+                host.unregisterMode = 'unregister';
+            } else {
+                host.unregisterMode = 'delete';
```

Comment:
Is there or will there ever be more than two "unregisterModes"?  If not, can we just use a boolean here?